### PR TITLE
Feat: Added a Lint checker

### PR DESCRIPTION
**Pull request by yokwejuste**
This PR is with respect to the issue #38 from the kanban board. You can review and merge when done.

@godsakani and @Sanix-Darker review on this is needed.

### CODE REVIEW

File: src/tracks/tracks.js [hunk 1]
```diff
@@ -5,44 +5,93 @@ export const track_oss = [
...
+        date: ' 02/05/2022',
+        fileUrl: 'https://raw.githubusercontent.com/muhammederdem/mini-player/master/mp3/9.mp3',
+    },
+    {
+        id: '13',
+        title: 'Full-Stack Development Guides',
+        podName: 'The OSS Community -',
+        date: ' 02/05/2022',
+        fileUrl: 'https://raw.githubusercontent.com/muhammederdem/mini-player/master/mp3/1.mp3',
+    },
```
Comment (Sanix-Darker) [inline src/tracks/tracks.js:96]: Why adding all these ??

File: src/App.js [hunk 2]
```diff
@@ -5,6 +5,7 @@ import 'bootstrap/dist/css/bootstrap.min.css';
 import './fonts/GoogleSans-Bold-v1.27.ttf';
 import './fonts/GoogleSans-Medium-v1.27.ttf';
 import './fonts/GoogleSans-Regular-v1.27.ttf';
+import 'bootstrap/dist/js/bootstrap.js';
```
Comment (Sanix-Darker) [inline src/App.js:8]: NO NO NO !
We don't want this !

**Review by Sanix-Darker — CHANGES_REQUESTED** [review]

File: src/tracks/tracks.js (hunk 1 above)
Comment (yokwejuste) [reply src/tracks/tracks.js:96]: Sir these lines was to simulate a situation where we could have a longer playlist to go through

**Comment by yokwejuste** [comment]
This PR contains some duplicate commits with respect to the PR #44 . For now I'll close it and correct some stuffs.

**Pull request closed without merging**